src/doc/src/guide/continuous-integration.md

```diff
@@ -49,6 +49,40 @@ breakage in nightly will not fail your overall build. Please see the
 [GitLab CI](https://docs.gitlab.com/ce/ci/yaml/README.html) for more
 information.
 
+### Github Actions
```

Comment:
This change seems unrelated?

Reply:
Oh indeed. Is it ok here or is a seperate pull request better?

Reply:
I think it's probably fine to be here.